aptos-move/aptos-vm-logging/src/log_schema.rs

```diff
@@ -52,7 +52,7 @@ impl AdapterLogSchema {
                 base_version: Some(base_version),
                 txn_idx,
             },
-            StateViewId::Miscellaneous => Self {
+            StateViewId::Miscellaneous | StateViewId::Replay => Self {
```

Comment:
see my comment below. depends on if you want speculative logging support, which you might want to enable.